version.properties

```diff
@@ -1,4 +1,4 @@
-#Thu Aug 10 15:36:20 UTC 2023
-dependency.coreSdk.version=1.0.3
-widgets.versionCode=66
-widgets.versionName=2.0.3
+#Wed Aug 30 16:18:14 UTC 2023
```

Comment:
Thursday?

Reply:
I cherry-picked those changes from development. And decided to leave them as is to avoid conflicts. I consider the date as a timestamp when the changes are ready for release, not the release itself.

Does it make sense to you?

Reply:
Do we need this line? Does it make any sense? Maybe just remove it?